tools/packaging/kata-deploy/helm-chart/kata-deploy/templates/kata-deploy.yaml

```diff
@@ -36,6 +56,73 @@ spec:
 {{- with .Values.nodeSelector }}
       nodeSelector:
 {{- toYaml . | nindent 8 }}
+{{- end }}
+{{- if index .Values "node-feature-discovery" "enabled" }}
```

Comment:
Hi @fidencio , if I'm not mistaken `kata-runtime kata-check` performs more checks than just virtualization support. Don't know if you plan to incorporate these in NFD too. Note for the future...

Reply:
That's what Zvonko mentioned in the last meeting as "image check". For sure we do want to do that in the future.